packages/webapp/src/routes/land/clients-email-address.js

```diff
@@ -1,13 +1,36 @@
 import constants from '../../utils/constants.js'
+import { validateEmail } from '../../utils/helpers.js'
 
 const handlers = {
   get: async (request, h) => {
-    return h.view(constants.views.CLIENTS_EMAIL_ADDRESS)
+    const email = request.yar.get(constants.redisKeys.CLIENTS_EMAIL_ADDRESS)
+    return h.view(constants.views.CLIENTS_EMAIL_ADDRESS, {
+      email
+    })
+  },
+  post: async (request, h) => {
+    const email = request.payload.email
+    const error = validateEmail(email, '#email')
+    if (error) {
+      if (error.err[0].text === 'Enter your email address') {
+        error.err[0].text = 'Enter email address'
```

Comment:
I don't have access to CD. Does this need to be different?

Reply:
Yes it's a bit of a fudge, but thought it would be better than over-complicating it, but it's because there is an email validation helper already used for several other routes that ask for the user's email, where as the copy for this ticket is not for the user's email.